UIKitTeamplate/Choose Coffee screen/OrderCofeeViewController.swift

```diff
@@ -0,0 +1,84 @@
+// OrderCofeeViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Протокол делегата выбора степени обжарки кофе
+protocol CoffeeRoastSelectionDelegate: AnyObject {
+    // Метод для передачи изображения выбранной степени обжарки
+    func didSelectRoastingImage(_ image: UIImage)
+    // Метод для передачи текста выбранной степени обжарки
+    func didSelectRoastText(_ text: String)
+}
+
+/// Экран с выбором позиций кофе для заказа
+final class OrderCofeeViewController: UIViewController {
+    var selectedRoastText: String?
+
+    // MARK: - Private Properties
```

Comment:
```suggestion
    // MARK: - Private Properties
```
не так марка это визуальные компоненты смотри в код стайл в вики